cdc/owner/scheduler_v1.go

```diff
@@ -310,6 +310,9 @@ func (s *oldScheduler) cleanUpFinishedOperations() {
 	for captureID := range s.state.TaskStatuses {
 		s.state.PatchTaskStatus(captureID, func(status *model.TaskStatus) (*model.TaskStatus, bool, error) {
 			changed := false
+			if status == nil {
```

Comment:
Is there any tests covers it?

Reply:
> Is there any tests covers it?

There are not enough test cases  for the callback functions like `PatchTaskStatus` in our code base. I will add some unit tests in new commits